sonar-slang-plugin/src/test/java/com/sonarsource/slang/kotlin/KotlinConverterTest.java

```diff
@@ -124,13 +105,17 @@ public void testFunctionDeclaration() {
     assertThat(functionDeclarationTree.modifiers()).hasSize(1);
     assertTree(functionDeclarationTree.returnType()).isIdentifier("Boolean");
     assertThat(functionDeclarationTree.formalParameters()).hasSize(2);
+    assertTree(functionDeclarationTree).hasParameterNames("a", "b");
     assertTree(functionDeclarationTree.body()).isBlock(LiteralTree.class);
 
-    FunctionDeclarationTree functionWithInternalModifier = (FunctionDeclarationTree) kotlin("internal fun function1(a: Int, b: String): Boolean = true");
+    FunctionDeclarationTree functionWithInternalModifier = (FunctionDeclarationTree) kotlin("internal fun function1(a: Int, c: String): Boolean = true");
     assertTree(functionWithInternalModifier.body()).isNotNull();
     assertThat(functionWithInternalModifier.modifiers()).hasSize(1);
+    assertTree(functionWithInternalModifier).hasParameterNames("a", "c");
 
     FunctionDeclarationTree functionWithPrivate = (FunctionDeclarationTree) kotlin("private fun function2() {}");
+    assertThat(functionWithPrivate.formalParameters()).isEmpty();
+    assertTree(functionWithPrivate).hasParameterNames();
```

Comment:
This assert `.hasParameterNames();` does not check anything special here. Since we already know `functionWithPrivate` is of type `FunctionDeclarationTree`